component-test/test_dbus_proxy.py

```diff
@@ -521,7 +521,7 @@ class TestProxyMethods(object):
             "direction": "*",
             "interface": "*",
             "object-path": "*",
-            "method": ["Method1", "Method3", "Method4", "Method2"]
+            "method": ["Method1", "Method3", "Method2", "Method4"]
```

Comment:
If that is the case then this test should be run twice with different order in the parameters. Otherwise, I do not feel it makes sense to push upstream.